.github/workflows/publish.yml

```
@@ -68,3 +68,35 @@ jobs:
           for tag in $(echo ${{ steps.build-push.outputs.metadata['image.name'] }} | tr ',' '\n'); do
             attach_eol_manifest $tag
           done
+
+  publish-extension:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # Needed to upload release assets
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'npm'
+      - name: Install extension dependencies
+        working-directory: ./extension
+        run: npm ci
+      - name: Build extension
+        working-directory: ./extension
+        run: npm run build
+      - name: Package extension
+        working-directory: ./extension
+        run: |
+          cd dist
+          zip -r ../playwright-mcp-extension-${{ github.event.release.tag_name }}.zip .
+          cd ..
+      - name: Upload extension to release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ./extension/playwright-mcp-extension-${{ github.event.release.tag_name }}.zip
+          asset_name: playwright-mcp-extension-${{ github.event.release.tag_name }}.zip
+          asset_content_type: application/zip
```

```suggestion
      - name: Upload extension to release
         env:
           GITHUB_TOKEN: ${{ github.token }}
         run: |
            gh release upload ${{github.event.release.tag_name}} ./extension/playwright-mcp-extension-${{ github.event.release.tag_name }}.zip
```

The action you are using is deprecated/unmaintained - I think something like this should work as well as per [cs](https://github.com/search?q=%22gh+release+upload%22+path%3A.github%2Fworkflows&type=code).